src/OrchardCore/OrchardCore.DisplayManagement/Entities/SectionDisplayDriver.cs

```diff
@@ -101,17 +101,17 @@ public virtual IDisplayResult Edit(TSection section)
             return null;
         }
 
-        public virtual Task<IDisplayResult> UpdateAsync(TModel model, TSection section, IUpdateModel updater, BuildEditorContext context)
+        public virtual Task<IDisplayResult> UpdateAsync(TModel model, TSection section, IUpdateModel updater, UpdateEditorContext context)
```

Comment:
@sebastienros Is there a reason why this SectionDisplayDriver uses BuildEditorContext on the UpdateAsync instead of UpdateEditorContext here. It is pretty useful to make a distinction. I tried this change and it still works where I tested it.